robottelo/utils/vault.py

```diff
@@ -19,6 +19,8 @@ class Vault:
 
     def __init__(self, env_file='.env'):
         self.env_path = robottelo_root_dir.joinpath(env_file)
+        self.envdata = self.env_path.read_text()
+        self.vault_enabled = re.findall('VAULT_ENABLED_FOR_DYNACONF=(.*)', self.envdata)[0]
```

Comment:
If `findall` finds nothing, it will throw a KeyError, am I correct?
If this is the case, it would be a good idea to handle it somehow.

I would like to see the following cases being handled:
1. there is no `.env` file in the robottelo root directory,
2. `.env` exists, but `VAULT_ENABLED_FOR_DYNACONF` is commented out,
3. `.env` exists, but `VAULT_ENABLED_FOR_DYNACONF` is not defined there.

Reply:
@ogajduse Resolved all possible combinations! Please revisit !

Reply:
Item 1 would  be very important to fix https://github.com/SatelliteQE/robottelo/issues/12816